daemons/dss-index/.chalice/config.json

```diff
@@ -6,7 +6,14 @@
   "stages": {
     "dev": {
       "api_gateway_stage": "dev",
-      "environment_variables": {}
+      "environment_variables": {
+        "DSS_DEPLOYMENT_STAGE": "dev",
+        "DSS_ES_ENDPOINT": "search-dss-index-dev-wc7kz2vuj7mpfjbxnp4ftplpuu.us-east-1.es.amazonaws.com",
+        "DSS_S3_BUCKET": "org-humancellatlas-dss-dev",
+        "DSS_GS_BUCKET": "org-humancellatlas-dss-dev",
+        "API_HOST": "hca-dss.czi.technology",
+        "DSS_SUBSCRIPTION_AUTHORIZED_DOMAINS": "chanzuckerberg.com ucsc.edu broadinstitute.org ebi.ac.uk human-cell-atlas-travis-test.iam.gserviceaccount.com broad-dsde-mint-dev.iam.gserviceaccount.com"
+      }
```

Comment:
Was this meant to be included? None of these variables are supposed to be hardcoded here, they are all passed in via `EXPORT_ENV_VARS_TO_LAMBDA` by https://github.com/HumanCellAtlas/data-store/blob/0f49e14d3085159307a17b6b808a6e96e2bc88ef/daemons/build_deploy_config.sh#L44-L46